File: MODUL 3/Python/PRAK305-2310817220011-ReginaSilvaMaharatini.py
num = int(input("Input\n"))
print("\nOutput")

hari = num // 86400
sisa1 = num % 86400
jam = sisa1 // 3600
sisa2 = sisa1 % 3600
menit = sisa2 // 60
detik = sisa2 % 60

if 0 <= num < 60:
    print(f"00:00:{detik:02d}")
elif 60 <= num <= 3599:
    print(f"00:{menit:02d}:{detik:02d}")
elif 3600 <= num <= 86399:
    print(f"{jam:02d}:{menit:02d}:{detik:02d}")
else:
    print(f"{hari} hari {jam:02d}:{menit:02d}:{detik:02d}")

File: MODUL 3/C/PRAK305-2310817220011-ReginaSilvaMaharatini.c
#include <stdio.h>

int main(){
    int num, hari, jam, menit, detik, sisa1, sisa2;
    printf("Input\n");
    scanf("%d", &num);
    printf("\nOutput");
    
    hari = num / 86400;
    sisa1 = num % 86400;
    jam =  sisa1 / 3600;
    sisa2 = num % 3600;
    menit =  sisa2 / 60;
    detik = num % 60;
    if(num >=0 && num < 60 ){
            printf("00:00:%d", detik);
    } else if(num >= 60 && num <= 3599){
            printf("00:%d:%d", menit, detik);
    } else if(num >= 3600 && num <= 86399){
          printf("%d:%d:%d", jam, menit, detik);
    } else{
        printf("%d hari %d:%d:%d", hari, jam, menit, detik);
    }
    return 0;
}